targets/arduino_101/src/main-zephyr.c

```diff
@@ -0,0 +1,132 @@
+/* Copyright 2016 Intel Corporation
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+#include <stdio.h>
+#include <string.h>
+#include <stdlib.h>
+
+#include <zephyr.h>
+#include <misc/printk.h>
+#include <misc/shell.h>
+
+#include "jerry.h"
+
+#if defined(CONFIG_STDOUT_CONSOLE)
+#include <stdio.h>
+#define PRINT           printf
+#else
+#include <misc/printk.h>
+#define PRINT           printk
+#endif
+
+static char *source_buffer = NULL;
+static unsigned char flags = 0;
+
+#define VERBOSE 0x01
+/**
+ * Jerryscript simple test loop
+ */
+int jerryscript_test() {
+    jerry_completion_code_t ret_code;
+    const char script[] =
+            "var test=0; " \
+            "for (t=100; t<1000; t++) test+=t; " \
```

Comment:
I will change it to "var" to be local in the loop. In some of my tests i was printing t after the loop to see how the global scope worked.